test/unstable/unstableConfigs.chpl

```diff
@@ -0,0 +1,12 @@
+config var myUserConfig = 0;
```

Comment:
While a general test of unstable configs is good, we want a test of the individual ones you've marked unstable, too

Reply:
Added tests for the flags grouped by module.